modular_zubbers/modules/quirks/neutral.dm

```diff
@@ -0,0 +1,80 @@
+// Bubber quirks
+
+/// 4/23 glowy_light value is null, failing to be assigned. dont know how to fix
+
+/datum/quirk/glowy
+	name = "Glowy"
+	desc = "You emit a small amount of light. Geiger counters might hate you."
+	value = 0
+	medical_record_text = "Patient's body contains moderate levels of emissive radiation. This seems mostly harmless to the surroundings."
+	icon = FA_ICON_BOLT_LIGHTNING
+	var/glowy_color = null /// Holds player's selected color
+	var/glowy_power = 1
+	var/glowy_range = 1
+	var/obj/effect/dummy/lighting_obj/moblight/glowy_light
+
+/datum/quirk/glowy/add_unique(client/client_source)
+	. = ..()
+	glowy_color = client_source?.prefs.read_preference(/datum/preference/color/glowy_color)
+
+/datum/quirk_constant_data/glowy
+	associated_typepath = /datum/quirk/glowy
+	customization_options = list(/datum/preference/color/glowy_color)
+
+/datum/quirk/glowy/add(mob/living/carbon/human/owner)
```

Comment:
This is of type client, and should not rename the original argument as that causes massive issues
```suggestion
/datum/quirk/glowy/add(client/client_source)
```